tests/Microsoft.Bot.Builder.Core.Extensions.Tests/ShowTyping_MiddlewareTests.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Adapters;
+using Microsoft.Bot.Schema;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Bot.Builder.Core.Extensions.Tests
+{
+    [TestClass]
+    public class ShowTyping_MiddlewareTests
```

Comment:
Consider adding a test that passes in invalid arguments and makes sure the exception is thrown.

Reply:
Tests added for both arguments